### ISSUE TITLE
Notification pushes SongSelect to the left causing unwanted padding

### ISSUE THREAD

**Issue by luaneko** [ppy/osu#3907]
**Describe your problem:** When notification panel is open in SongSelect screen, it adds unwanted padding on the side, making the scrollbar *float*.

**Screenshots or videos showing encountered issue:**
Before:
![before](https://i.imgur.com/Nu9fmML.jpg)
After:
![after](https://i.imgur.com/oPq1a6X.png)

**osu!lazer version:** 2018.1219.0

**Comment by phosphene47** [comment]
imo the scrollbar should always stick to the side, like a scrollbar. it feels wrong otherwise.
also, you can the the rounded corners of the active beatmapset.

**Comment by peppy** [comment]
Yeah, I'm aware of both of these but still don't strongly disagree with it being that way for now.

**Comment by bdach** [comment]
Not really relevant anymore because the notification panel has an opaque background now.
